src/main/webapp/AddSchoolData.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>School Form</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <p>Add A Course Below</p>
+    <form action="/add-info" method="POST">
+        <p>School Name</p>
+        <textarea name="school-name">SchoolName</textarea>
+        <br><br>
+        <p>Course Name</p>
+        <textarea name="course-name">CourseName</textarea>
+        <br><br>
+        <p>Term</p>
+        <textarea name="term">Term</textarea>
+        <br><br>
+        <p>Units</p>
+        <input type="number" name="units" min="1" max="24">
+        <br><br>
```

Comment:
Consider switching out your bracket tags for padding and margin in CSS (Luke made a comment of this in my PR)